docker-compose-generator/docker-fragments/opt-add-tor.yml

```diff
@@ -41,6 +41,13 @@ services:
     volumes:
       - "tor_servicesdir:/var/lib/tor/hidden_services"
 
+  prestashop:
```

Comment:
Remove this if you don't want to expose over tor

Reply:
Oh yes of course 🤦

Reply:
Oh yes, of course 🤦